src/test/java/jenkins/plugins/logstash/persistence/SyslogDaoTest.java

```diff
@@ -0,0 +1,18 @@
+package jenkins.plugins.logstash.persistence;
+
+import org.junit.Test;
+
+public class SyslogDaoTest {
+  SyslogDao dao;
+  
+  @Test
+  public void send() throws Exception {
+    String data = "{ 'junit': 'SyslogDaoTest' }";
+    dao = createDao("localhost", 514, null, null, null);
+    dao.push(data);
```

Comment:
This isn't a very useful test. You could comment-out all of the code in the push method and this test would still pass.

I'd recommend mocking out the messageSender object and at a minimum verify that the right payload was sent. Take a look at the other DAOs and their tests for examples.